libyang/schema.py

```diff
@@ -681,6 +681,12 @@ def all_patterns(self) -> Iterator[Tuple[str, bool]]:
         else:
             yield from self.patterns()
 
+    def require_instance(self) -> Optional[bool]:
+        if not self.cdata or self.cdata.basetype != self.LEAFREF:
+            return None
```

Comment:
self.cdata is guaranteed. It is self.cdata_parsed that is not.
You can safely do "if self.cdata.basetype != self.LEAFREF:"

Reply:
Done